e2e/cypress/mockApis/incentivesApi.js

```diff
@@ -0,0 +1,23 @@
+const { stubFor } = require('./wiremock');
+
+const stub = (urlPattern, jsonBody) =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern,
+    },
+    response: {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json;charset=UTF-8',
+      },
+      jsonBody,
+    },
+  });
+
+const stubIncentives = incentives =>
```

Comment:
Created this before realising that all cypress tests have been moved to the `ignored` directory and the `run_e2e_tests` CircleCI job has been commented out. But at least we have a stub if they're ever switched back on